cleverhans_tutorials/mnist_tutorial_cw.py

```diff
@@ -237,6 +237,13 @@ def mnist_tutorial_cw(train_start=0, train_end=60000, test_start=0,
 
 
 def main(argv=None):
```

Comment:
Could we factor this into one function that is called by several tutorials?

We could define it in `cleverhans_tutorials/__init__.py` and then just put

`
from cleverhans_tutorials import check_installation
check_installation()
` in each tutorial.